tutorials/scripting/gdscript/gdscript_advanced.rst

```diff
@@ -364,9 +364,9 @@ The range() function can take 3 arguments:
 
 ::
 
-    range(n) # Will go from 0 to n-1.
-    range(b, n) # Will go from b to n-1.
-    range(b, n, s) # Will go from b to n-1, in steps of s.
+    range(n) # Will count from 0 to n in steps of 1. The parameter n is exclusive.
+    range(b, n) # Will count from b to n in steps of 1. The parameters b and n are inclusive and exclusive, respectively.
```

Comment:
For non-native english speakers, this is easier to understand (line 368)
`range(b, n) # Will count from b to n in steps of 1. The parameter b is inclusive. The parameter n is exclusive.`
Same applies for line 369